hack/templates/related-images/related_images.template

```diff
@@ -0,0 +1,21 @@
+  relatedImages:
+    - name: nvidia-network-operator-init-container
```

Comment:
The NO operator itself is not here because `hack/release.yaml` does not contain its version.
Should we add it there?

Reply:
if you manage to test oc mirror and it mirrors the operator without it then no need imo.
else lets add it (according to provided user input i assume)